src/mon/NVMeofGwMon.cc

```diff
@@ -429,76 +446,76 @@ bool NVMeofGwMon::preprocess_beacon(MonOpRequestRef op)
   return false;
 }
 
-
-bool NVMeofGwMon::prepare_beacon(MonOpRequestRef op)
-{
-  auto m = op->get_req<MNVMeofGwBeacon>();
+bool NVMeofGwMon::prepare_beacon(MonOpRequestRef op) {
+    auto m = op->get_req<MNVMeofGwBeacon>();
 
   dout(20) << "availability " <<  m->get_availability()
-	   << " GW : " << m->get_gw_id()
-	   << " osdmap_epoch " << m->get_last_osd_epoch()
-	   << " subsystems " << m->get_subsystems() << dendl;
+        << " GW : " << m->get_gw_id()
+        << " osdmap_epoch " << m->get_last_osd_epoch()
+        << " subsystems " << m->get_subsystems() << dendl;
```

Comment:
This chunk changes appears to be an accidental revert of my cleanup PR (changes the indent but doesn't change the content).  Please check over this entire PR for other examples and remove them.